### PR TITLE
Fix Bitbucket's main logo, button highlights and shadows

### PR DESCRIPTION
**Pull request by Neptilo**

### CODE REVIEW

**Comment by Neptilo** [comment]
Note: didn't manage to use `var(--darkreader-neutral-text)` in cobination with an alpha value.

**Review by AndrewDaGuy — CHANGES_REQUESTED** [review]
Pretty sure Guested does not like hard coded colours

**Comment by Neptilo** [comment]
@AndrewDaGuy Sure. But have you read my comment?

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -1293,6 +1293,19 @@ span[data-testid="file-tree-file__comments"] > span {
...
+[type="button"]:not(.css-1l4j2co, .css-1h2061w, [aria-expanded]):hover {
+    background-color: rgba(159, 179, 213, 0.48) !important;
+}
+.ak-navigation-resize-button {
+    box-shadow: rgba(109, 127, 157, 0.48) 0px 0px 0px 1px,
+                rgba(109, 127, 157, 0.48) 0px 2px 4px 1px;
+}
+
+IGNORE INLINE STYLE
+svg *
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:1308]: Better specify of the svg would be needed as thing _could_ break other svg's.

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -1293,6 +1293,19 @@ span[data-testid="file-tree-file__comments"] > span {
 span.code {
     background-color: rgba(255, 247, 218, 0.08) !important;
 }
+[type="button"]:not(.css-1l4j2co, .css-1h2061w, [aria-expanded]) {
+    background-color: rgba(159, 179, 213, 0.24) !important;
+}
+[type="button"]:not(.css-1l4j2co, .css-1h2061w, [aria-expanded]):hover {
+    background-color: rgba(159, 179, 213, 0.48) !important;
+}
+.ak-navigation-resize-button {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:1302]: Where can I find the resize-button?

File: src/config/dynamic-theme-fixes.config [hunk 3]
```diff
@@ -1293,6 +1293,19 @@ span[data-testid="file-tree-file__comments"] > span {
 span.code {
     background-color: rgba(255, 247, 218, 0.08) !important;
 }
+[type="button"]:not(.css-1l4j2co, .css-1h2061w, [aria-expanded]) {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:1296]: You can exclude the `.css-1l4j2co, .css-1h2061w,` as they pretty sure would be randomized when bitbucket deploys an new version.

File: src/config/dynamic-theme-fixes.config [hunk 4]
```diff
@@ -1293,6 +1293,19 @@ span[data-testid="file-tree-file__comments"] > span {
...
+    background-color: rgba(159, 179, 213, 0.24) !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:1297]: Use something along the lines
```css
background-color: ${rgba(46, 66, 100, 0.24)} !important;
```

File: src/config/dynamic-theme-fixes.config [hunk 5]
```diff
@@ -1293,6 +1293,19 @@ span[data-testid="file-tree-file__comments"] > span {
...
+}
+[type="button"]:not(.css-1l4j2co, .css-1h2061w, [aria-expanded]):hover {
+    background-color: rgba(159, 179, 213, 0.48) !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:1300]: Use something along the lines
```CSS
background-color: ${rgba(46, 66, 100, 0.48)} !important;

File: src/config/dynamic-theme-fixes.config [hunk 6]
```diff
@@ -1293,6 +1293,19 @@ span[data-testid="file-tree-file__comments"] > span {
 span.code {
     background-color: rgba(255, 247, 218, 0.08) !important;
 }
+[type="button"]:not(.css-1l4j2co, .css-1h2061w, [aria-expanded]) {
+    background-color: rgba(159, 179, 213, 0.24) !important;
+}
+[type="button"]:not(.css-1l4j2co, .css-1h2061w, [aria-expanded]):hover {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:1299]: You can exclude the `.css-1l4j2co, .css-1h2061w,` as they pretty sure would be randomized when bitbucket deploys an new version.

File: src/config/dynamic-theme-fixes.config (hunk 2 above)
Comment (Neptilo) [reply src/config/dynamic-theme-fixes.config:1302]: If you are logged in, it is the arrow button that appears when you hover over the left panel:
image

File: src/config/dynamic-theme-fixes.config (hunk 4 above)
Comment (Neptilo) [reply src/config/dynamic-theme-fixes.config]: Thanks for the tip! I shall copy the exact initial color codes then.

**Review by Gusted — APPROVED** [review]
Thank you.

**Pull request merged**